CondTools/Ecal/python/EcalMustacheSCParametersMaker_cfg.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I think this file might belong better in /test/, it doesn't seem like something you would want to import from other configs.

Reply:
It seemed to be the right place. There are many similar looking cfg.py files in the python directory. But I can move it to /test as well. Does not make a difference as long as we remember where to find it.

Reply:
My understanding (which is far from complete) has been that we put in /python/ general "building blocks" for configuration, whereas in /test/ various standalone runnable configs.

Doing a check in the reco packages at least (CondTools is not reco), I see that cms.Process is very rarely defined in the /python/ dir. Perhaps there is a different approach in CondTools or AlCa packages in general.

```
$ find . -wholename "./Reco*/*/python/*" -name "*.py" -exec grep -H "cms\.Process" {} \; | wc -l
5
$ find . -wholename "./Reco*/*/test/*" -name "*.py" -exec grep -H "cms\.Process" {} \; | wc -l
317
```

Reply:
I guess this has not been handled in the same way all the time. And different packages might handle things different on top of it. EDM modules are also not always in the plugins directory. Anyway, I have moved the cfg files to the test directory now where they are fine as well.